packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -176,20 +176,23 @@
                     <oui-tile-description>
                         <oui-tile-description>
                             <span
-                                ng-if="$ctrl.getCurrentFlavor().minDiskSize === $ctrl.getCurrentFlavor().maxDiskSize"
-                                data-translate="pci_databases_general_information_node_type"
+                                data-translate="pci_databases_general_information_node_type_cpu_memory"
                                 data-translate-values="{
                                 coreNumber: $ctrl.getCurrentFlavor().core,
                                 memorySize: ($ctrl.getCurrentFlavor().memory | cucBytes:2:false:'GB'),
+                            }"
+                            ></span>
+                            <span
+                                ng-if="$ctrl.getCurrentFlavor().maxDiskSize > 0 && $ctrl.getCurrentFlavor().minDiskSize === $ctrl.getCurrentFlavor().maxDiskSize"
```

Comment:
We're starting to get too much logic in the template here, maybe using some utility function on a Flavor model could help

Reply:
Hi @marie-j , I have added the utility functions in flavor model as you suggested: https://github.com/ovh/manager/pull/5984/commits/27a6269da94f31d88605a6adb87cf24ab9f974d4